### PR TITLE
Remove quickstarts from docs

### PR DESCRIPTION
**Pull request by AaronCrawfis**
This PR removes quickstarts, migrating the existing guides into how-to guides and tutorials.

Note that tutorials still need to be documented and contribution guides need to be written, as part of a follow-up PR.

### CODE REVIEW

File: docs/content/tutorials/tutorial-dapr/index.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 ---
 
-This quickstart will teach you:
+This how-to guide will teach you:
 
 - How to use Radius to deploy a Dapr microservices sample application for an online shop
 - How [Dapr and Radius]({{< ref dapr-resources >}}) seamlessly work together
 
-For more details on the app and access to the source code, visit the `quickstarts/dapr` directory in the [samples repo](https://github.com/project-radius/samples). _For access to the project-radius GitHub org, please complete and submit [this form](https://aka.ms/ProjectRadius/GitHubAccess)._
+For more details on the app and access to the source code, visit the `tutorials/dapr` directory in the [samples repo](https://github.com/project-radius/samples). _For access to the project-radius GitHub org, please complete and submit [this form](https://aka.ms/ProjectRadius/GitHubAccess)._
```
Comment (willtsai) [inline docs/content/tutorials/tutorial-dapr/index.md:17]: Does the `samples/quickstarts/...` need to be renamed as well?

File: docs/content/getting-started/reference-apps/aws-sqs/_index.md
Comment (willtsai) [inline docs/content/getting-started/reference-apps/aws-sqs/_index.md:1]: I think this file name is causing issues with how the side bar renders and resulting in the S3 page being blank:

image

**Review by willtsai — CHANGES_REQUESTED** [review]
some general feedback:
1. I feel that the `reference apps` section should be nested under `tutorials` given the similarity in the intent of the content
2. I'm torn on whether the `languages/bicep` section should be nested under `getting started` as it's a pretty nuanced topic, perhaps it fits better under `concepts`?

**Comment by AaronCrawfis** [comment]
> I feel that the reference apps section should be nested under tutorials given the similarity in the intent of the content

Reference apps are also going to be removed in an upcoming PR, with eShop becoming a tutorial. Still working out where to put the others (or remove them). But that will be a follow up PR.

> I'm torn on whether the languages/bicep section should be nested under getting started as it's a pretty nuanced topic, perhaps it fits better under concepts?

The Bicep page will be converted into a how-to guide (or overview) under deploying apps in a follow up PR. Wanted to focus on just removing quickstarts in this PR.

File: docs/content/getting-started/reference-apps/aws-sqs/_index.md
Comment (AaronCrawfis) [reply docs/content/getting-started/reference-apps/aws-sqs/_index.md:1]: That's an existing issue that exists already on docs.radapp.dev. I can get that fixed when I go in to remove the reference apps in a follow up PR

File: docs/content/tutorials/tutorial-dapr/index.md (hunk 1 above)
Comment (AaronCrawfis) [reply docs/content/tutorials/tutorial-dapr/index.md:17]: Yes the container image names will need to be updated, but that will require a PR into the samples repo and redoing the CI/CD systems. Opened an Issue to track as part of a follow up PR: https://github.com/project-radius/samples/issues/462

**Comment by AaronCrawfis** [comment]
Closing in favor of #690

**Pull request closed without merging**